packages/constants/src/logLevels.js

```diff
@@ -0,0 +1,6 @@
+export const TRACE = 'trace'
```

Comment:
Realizing that this should probably be lower case, and the value should be the numeric log level rather than a string, e.g. https://github.com/winstonjs/winston#logging

Reply:
Refactored to reflect that need, added test based on that spec.

Reply:
See also https://github.com/winstonjs/winston#logging-levels. Though I'm not sure whether you can pass the number to Winston. In general, you'd log to the `warn` level using `logger.warn` for instance, with Winston, correct?

Reply:
I'll check to see if you can pass the number to winston of if it requires the string (if it requires the string then 😬 since I think we really do need the constant to be linked to a number too).

I would tend to use logger.warn(...) but I'm under the impression that that's just syntactic sugar to logger.log('warn',...)

Reply:
Looks like you'd tell Winston about the custom levels at [creation](https://github.com/winstonjs/winston#using-custom-logging-levels).

Just tried `logger.log(1, ...)` and `logger.log('trace', ...)` and got `Unknown logger level`. I still got `logger.trace is not a function` when I did `logger.trace(...)` even after defining it as a custom log level, though  `logger.log('trace', ...)` then worked.

I looked at a bunch of this when troubleshooting the Jest/Winston interaction.

Reply:
Got it!

So, I'm silly -- I just realized that it's totally fine for the log level strings to be the constant, and to have the winston registration itself assign things like value and color.  We don't need winston attributes to be part of the base constant!

Revising now back to how I had it originally.